common/config/rush/command-line.json

```diff
@@ -1,201 +0,0 @@
-/**
- * This configuration file defines custom commands for the "rush" command-line.
- * For full documentation, please see https://rushjs.io
- */
-{
-  "$schema": "https://developer.microsoft.com/json-schemas/rush/v5/command-line.schema.json",
-
-  /**
-   * Custom "commands" introduce new verbs for the command-line.  To see the help for these
-   * example commands, try "rush --help", "rush my-bulk-command --help", or
-   * "rush my-global-command --help".
-   */
-  "commands": [
-    {
-      /**
-       * (Required) The name that will be typed as part of the command line.  This is also the name
-       * of the "scripts" hook in the project's package.json file.
-       * The name should be comprised of lower case words separated by hyphens.
-       */
-      "name": "code-style",
```

Comment:
Does lerna have an equivalent central location for commands and options like this?